### ISSUE TITLE
Card number validation state not updated after pasting the number via ctrl+v

### ISSUE THREAD

**Issue by rebendajirijr** [braintree/braintree-web-drop-in#613]
### General information

* SDK/Library version: [1.22.0](https://js.braintreegateway.com/web/dropin/1.22.0/js/dropin.min.js) (used in [angular wrapper library](https://github.com/htmldriven/ngx-braintree))
* Environment: Sandbox (didn't try it in production)
* Browser and OS: Windows, Firefox 74.0/Chrome 80.0.3987.149 (Official Build) (64-bit)

#### Steps to reproduce:
1. Choose payment by card.
![image](https://user-images.githubusercontent.com/3431185/77565174-231d7580-6ec4-11ea-97a6-5cc8b41dcc53.png)
2. Enter some invalid card number (in this example **340000000001007**):
![image](https://user-images.githubusercontent.com/3431185/77564431-003e9180-6ec3-11ea-82d7-9be30cb4bc65.png)
3. Paste a valid card number using the `ctrl+v` shortcut (in this example **5200000000001005**):
![image](https://user-images.githubusercontent.com/3431185/77564543-2ebc6c80-6ec3-11ea-9b28-76df8d4672ba.png)

#### Expected result:
Validation error should disappear when valid card number pasted via `ctrl+v` shortcut.

#### Actual result:
Validation error remains visible. The validation error disappears only after manually changing the input value by typing (as you can see in the picture below).
![image](https://user-images.githubusercontent.com/3431185/77564865-abe7e180-6ec3-11ea-92ac-dd6e2cbaaf88.png)

**Comment by crookedneighbor** [comment]
Thanks for the report. 

This is happening because we were using custom logic to check for card type support, rather than relying on the raw sdk (since at the time, the raw sdk had no notion of supported card types)

We're currently working on version 2 of Drop-in, and it's [addressed there](https://github.com/braintree/braintree-web-drop-in/blob/7812c25b722b14d5dfd2005e90408bdbea019437/src/views/payment-sheet-views/card-view.js#L304):

![cards](https://user-images.githubusercontent.com/2916945/77580025-47b82400-6ea9-11ea-905b-884facdee20b.gif)

Since this isn't an urgent bug, I don't think we'll prioritize getting this fixed in v1, since v2 is on the horizon. Will close this issue once v2 is released.

**Comment by studios** [comment]
If I can ask one question: I was searching in the Drop-in UI Developer guides; however, I can't find a way how to remove some of the logos of cards visible on the right-side of the Pay with card form (see highlight in the image below). I'm asking this because we accept only Visa and Mastercard (and no Amex or others), and by having all logos there creates confusion and generates unsuccessful payments. Many thanks.

![Braintree - cards](https://user-images.githubusercontent.com/473007/104808773-bc699f80-57e8-11eb-8bcd-e6350a47c528.png)

**Comment by crookedneighbor** [comment]
@studios that's a separate issue from this, I've opened a new issue for you here: https://github.com/braintree/braintree-web-drop-in/issues/688

**Comment by matthewp672261** [comment]
we're having the same issue where validation of the card number (and sometimes the zip code) is not happening when filling saved card info. the fields remain yellow and the 'paymentMethodRequestable' is not subsequently triggered. some customers seem to believe our site is broken and bounce because of this. editing the fields by a single character resolves the validation issue. we've updated to the latest version as of this post (v1.33.2)

the original issue is > 2 years old. is there any guidance on how to handle this situation? thanks in advance!

**Comment by oscarleonnogales** [comment]
We've released a fix for this as of version 1.38.0

You can selectively disable different card brands by setting the appropriate fields in the `card.overrides.fields.number.supportedCardBrands` object when creating the Braintree dropin instance, disabling the card brands this way will now also prevent the respective icon from rendering.

Example shown below.

```
braintree.dropin.create({
        authorization: 'your_token',
        container: '#dropin-container',
        card: {
          overrides: {
            fields: {
              number: {
                supportedCardBrands: {
                  visa: true,
                  mastercard: true,
                  "american-express": false
                },
              }
            }
          }
        }
      })
```
